application/src/ext/java/org/opentripplanner/ext/siri/updater/azure/AbstractAzureSiriUpdater.java

```diff
@@ -64,14 +97,14 @@ public abstract class AbstractAzureSiriUpdater implements GraphUpdater {
   protected final int timeout;
 
   public AbstractAzureSiriUpdater(SiriAzureUpdaterParameters config) {
-    this.configRef = config.configRef();
-    this.authenticationType = config.getAuthenticationType();
-    this.fullyQualifiedNamespace = config.getFullyQualifiedNamespace();
-    this.serviceBusUrl = config.getServiceBusUrl();
-    this.topicName = config.getTopicName();
-    this.dataInitializationUrl = config.getDataInitializationUrl();
+    this.configRef = Objects.requireNonNull(config.configRef(), "configRef must not be null");
```

Comment:
It's of course a good idea to reject null values here, but you can also remove the default values (`null` :man_shrugging: ): https://github.com/opentripplanner/OpenTripPlanner/blob/f4bfedfc41d3b92df45927a72edcab28624510d6/application/src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/azure/SiriAzureUpdaterConfig.java#L19-L95

It seems that most of them are not optional.

Reply:
The Azure config also contains some questionable hyphenation.

Reply:
Yeah, the hyphenation is a bit weird. But I think we can leave it since it's a sandbox feature and it will only cause us unnecessary work if we have to update all our configuration files

Reply:
left as is